dev/benchmarks/complex_layout/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous
```

Comment:
We're pretty inconsistent about when we add `.gitignore` files in these examples.

@jmagman will know whether they're meant to be checked in or not. This probably suggests we should have a lint if they're not meant to be checked in (or I should just remember 😉).